src/changelog/.2.x.x/simplify_configuration_after_bnd_7_update.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to you under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<entry xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns="http://logging.apache.org/log4j/changelog"
+       xsi:schemaLocation="http://logging.apache.org/log4j/changelog https://logging.apache.org/log4j/changelog-0.1.3.xsd"
+       type="updated">
+  <description format="asciidoc">
+    Simplify BND configuration after upgrade from version 6.4.1 to 7.0.0.
```

Comment:
Please note that this is of type `changed`, not `updated`. I guess you play this trick to get a patch release out. Let's not trick anyone, but be transparent. (I don't have energy to argue about patch-vs-minor anymore.) Please use `updated` only for, as its name implies and we agreed on, dependency updates.

```suggestion
       type="changed">
  <description format="asciidoc">
    Simplify BND configuration after upgrade from version `6.4.1` to `7.0.0`
```

Reply:
Technically it **is** a dependency update: it should have **no** effect on the user. The only reason I add it to the changelog is that these kind of things _might_ contain errors and I want to be plenty transparent about the "no-op" changes that occurred.

Maybe it should be a "fixed" change, since it actually fixes the usage of Conversant Media Disruptor.